src/main/java/Main.java

```diff
@@ -1,8 +1,16 @@
 package main.java;
 
+import java.util.Arrays;
+import java.util.List;
+
 public class Main {
 
     public static void main(String[] args) {
-        System.out.println("Hello World!");
+        List<String> list = Arrays.asList("hello","world");
+        String str1 = "";
+        for(String str : list){
+            str1 += str;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use a StringBuilder instead. ([squid:S1643](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS1643))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=1&issues=AWQNxUEJ3e-qcckjJm6I&open=AWQNxUEJ3e-qcckjJm6I)

[ISSUE_KEY]: AWQNxUEJ3e-qcckjJm6I